vb-fe/src/_store/system/SystemTypes.ts

```diff
@@ -4,14 +4,20 @@ export enum SystemConstants {
   SUCCESS = "SUCCESS",
   FAILURE = "FAILURE",
   LOGIN = "LOGIN",
-  CREATE_ROOM_MODAL = "CREATE_ROOM_MODAL",
+  SET_ROOM_MODAL = "SET_ROOM_MODAL",
   EXPAND_BOTTOM_SHEET = "EXPAND_BOTTOM_SHEET",
   SOCKET = "SOCKET",
   RETRY = "RETRY",
+  EDIT = "EDIT",
+  CREATE = "CREATE",
   INITIALIZED = "INITIALIZED",
   SOCKETS_CONNECTED = "SOCKETS_CONNECTED",
 }
 
+export type RoomModalType =
```

Comment:
change to interface. we have used only interfaces throughout the FE, so lets just keep it consistent